completions/bash/podman

```diff
@@ -1524,6 +1524,57 @@ _podman_info() {
     esac
 }
 
+_podman_image_umount() {
+     _podman_image_unmount
+}
+
+_podman_image_unmount() {
+    local boolean_options="
+	--all
+	-a
+	--help
+	-h
+	--force
+	-f
+    "
+    local options_with_args="
+    "
+
+    local all_options="$options_with_args $boolean_options"
+    case "$cur" in
+	-*)
+	    COMPREPLY=($(compgen -W "$boolean_options $options_with_args" -- "$cur"))
+	    ;;
+	*)
+	    __podman_complete_container_names
```

Comment:
This seems wrong, the names to complete here are not container names, but existing image mounts (i.e. the output of `podman image mount` filtered by ID).

Reply:
Or `__podman_complete_images` filtered by those that are actually mounted.

Reply:
FIxed